contrib/mesos/docs/issues.md

```diff
@@ -37,27 +121,93 @@ Kubernetes clusters assume a custom Docker networking configuration that assigns
 
-There are two work-arounds to this problem:
-* Restart the framework and it should terminate the orphaned tasks.
-* Adjust the value of `executor_shutdown_grace_period` to something greater than 3 seconds.
+Some factors that influence when pods are scheduled by k8s-mesos:
+- availability of a resource offer that "fits" the pod (mesos master/slave);
+- scheduler *backoff* (to avoid busy-looping) during pod scheduling (k8s-mesos scheduler)
+
+We have tried to mitigate the second item by cancelling the backoff period if an offer arrives that fits a pod-in-waiting.
+However, there is nothing that the scheduler can do if there are no resources available in the cluster.
+
+That said, the current scheduling algorithm is somewhat naive: it makes **no attempts to pack multiple pods into a single offer**.
+This means that each pod launch requires an independent offer.
+In a small cluster resource offers do not arrive very frequently.
+In a large cluster with a "decent" amount of free resources the arrival rate of offers is expected to be much higher.
+
+The slave on each host announces offers to Mesos periodically.
+In a single node cluster only a single slave process is advertising resources to the master.
+The master will pass those along to the scheduler, at some interval and level of 'fairness' determined by mesos.
+That scheduler will pair each resource offer with a pod that needs to be placed in the cluster.
+Once paired, a task is launched to instantiate the pod.
+The used resources will be marked as consumed, the remaining resources are "returned" to the cluster and the scheduler will wait for the next resource offer from the master... and the cycle repeats itself.
+This likely limits the scheduling throughput observable in a single-node cluster.
+
+We are currently running some benchmarks on the scheduling algorithm to establish some baselines, and are definitely thinking about ways to increase scheduling throughput- including scheduling multiple pods per offer.
+
+#### Runtime Configuration
+
+- mesos: `--offer_timeout` : Duration of time before an offer is rescinded from a framework.
+  This helps fairness when running frameworks that hold on to offers, or frameworks that accidentally drop offers.
+  ([via](http://mesos.apache.org/documentation/latest/configuration/))
+- k8s-mesos `--scheduler-config` : An ini-style configuration file with low-level scheduler settings.
+  See `offer-ttl`, `initial-pod-backoff`, and `max-pod-backoff`.
+  ([via](https://github.com/kubernetes/kubernetes/blob/master/contrib/mesos/pkg/scheduler/config/config.go))
+
+What's not configurable, but perhaps should be, are the mesos "filters" that we include when declining offers that we cannot match to pods within the config'd `offer-ttl` (see https://github.com/apache/mesos/blob/0.25.0/include/mesos/mesos.proto#L1165). I think we hardcode the `refuse_seconds` value to 5s.
```

Comment:
Yeah and the "I think ..." is strange. Let's check whether we hardcoded it and make it a fact here.